models/logs.js

```diff
@@ -47,12 +49,39 @@ const fetchLogs = async (query, param) => {
     });
     return logs;
   } catch (err) {
-    logger.error("Error in adding log", err);
+    logger.error("Error in fetching logs", err);
+    throw err;
+  }
+};
+
+/**
+ * Fetches member cache Self logs
+ *
+ * @param userId { String }: Unique ID of the User
+ */
+const fetchMemberCacheLogs = async (id) => {
+  try {
+    const logsSnapshot = await logsModel
+      .where("type", "==", logType.CLOUDFLARE_CACHE_PURGED)
+      .where("timestamp", ">=", getLast24HourTime(admin.firestore.Timestamp.fromDate(new Date())))
```

Comment:
Instead of `getLast24HourTime`, can't we use `getTimeInSecondsAfter({ days: -1 })` directly here?

Replying to - https://github.com/Real-Dev-Squad/website-backend/pull/632/files#r918011617

Reply:
Ohh ok you were saying that we can use the existing util function. I have to see if we can. I will get back here after trying out this

Reply:
Actually we wanted to get the timestamp in the firestore format but `getTimeInSecondsAfter({ days: -1 })`  function returns the time in milliseconds, also I tried with `days: -1` but function is not designed that way and it returns `NaN`

Reply:
ok